engine/minskyTensorOps.cc

```diff
@@ -186,8 +186,9 @@ namespace minsky
   TensorOpFactory::TensorOpFactory()
   {
     registerType<TimeOp>(OperationType::time);
-    registerOps<MultiWireBinOp, OperationType::add, OperationType::log>(*this);
-    registerOps<TensorBinOp, OperationType::log, OperationType::copy>(*this);
+    registerOps<MinskyTensorOp, OperationType::euler, OperationType::pi>(*this);
```

Comment:
The second argument is exclusive, not inclusive, like a container range (begin(), end()). So the second argument should be whatever comes after pi.

Reply:
Thanks. Fixing it now.